README.md

```diff
@@ -66,6 +66,25 @@ end
 
 ## Advanced Usage
 
+### Send API enhanced
+
+```ruby
+client = Courier::Client.new "your-auth-token" # or set via COURIER_AUTH_TOKEN env var
+res = client.send_message({
+    message => {
+      template => "my-template",
+      to => {
```

Comment:
This is more elaborate than this, Are we hoping to cover that part in public-facing docs?